python/pyspark/serializers.py

```diff
@@ -359,6 +359,24 @@ def __repr__(self):
         return "ArrowStreamPandasSerializer"
 
 
+class InterleavedArrowReader(object):
+
+    def __init__(self, stream):
+        import pyarrow as pa
+        self._schema1 = pa.read_schema(stream)
```

Comment:
I wanted to read these also using the message reader but for some reason pa.read_schema(self_reader.read_next_message()) didn't work.